_posts/2017-11-08-cost-of-drupal-7.md

```diff
@@ -0,0 +1,180 @@
+
+To quote Angie Byron for the thousandth time ([full video here](https://youtu.be/wWnhfTSkmoU?t=386)):
+
+> For people who are classically trained or have experience in other languages, Drupal 8 is going to make a lot more sense to them than Drupal 7 did. We're just falling more in line with what the larger people are doing… within the broader PHP community.
+
+### WIIFM? Cost savings.
+
+The active decision to upgrade or passive indecision to wait both have cost implications. Perhaps this is the most useful section for readers whose primary responsibilities aren't technical.
+
+Continuing to invest in Drupal 7 (or earlier) can be costly in ways that may not be abundantly apparent on the surface. For most organizations who work with an agency, custom development is where a brunt of the efforts are spent, and therefore is the primary cost driver. "Custom development" occurs when the functionality a client requests is either not freely available on the open-source market, or the agency is unaware of its existence, and a developer will write code for the specific use case to "extend" the out-of-the-box functionality. The [80-20 rule](https://en.wikipedia.org/wiki/Pareto_principle) applies well to software development in Drupal; roughly 20% of the functionality a client requests accounts for 80% of the effort of a project since that 20% must be built from scratch. When site owners request various functionality, it can be difficult for them to differentiate what may constitute custom development efforts vs. freely available from the contributed community. Given the high effort of customization and related technical debt accumulated, we advise site owners to request a high degree of transparency to understand what requires custom development when establishing project budgets. This way, the organization can do a cost/benefit analysis on a granular, per-feature basis. The goal for developers should be to always start with exploring what already-made wheels are out there for the turning before crafting their own. Be wary of alternative thinking. Yet, as extensible and rich the Drupal community is, nearly all of our engagements require customization.
```

Comment:
> we advise site owners to request a high degree of transparency

I see what you're saying now, but at first this made no sense because it seemed like you're saying that we advise our own clients to request transparency from us. Maybe change to:

> site owners should request...